### DISCUSSION TITLE
ScreenModel on shared module

### DISCUSSION THREAD

**Discussion by jfmargar** [adrielcafe/voyager#367] [Q&A]
Hi, I'm working on a kmm project and I want to share the viewModels in shared module. View layer are native: SwiftUI on iOS and JetpackCompose for android. I read de doc and see that ScreenModel is android independent

Is it possible do it? When I add voyager dependencies on commonMain.dependencies the iOS App not run

**Comment by yuanfuautotrader** [comment]
Same issue here, if you found any solutions?
